pandas_gbq/tests/test_gbq.py

```diff
@@ -19,9 +19,9 @@
 import pandas.util.testing as tm
 from pandas.compat.numpy import np_datetime64_compat
 
-PROJECT_ID = None
-PRIVATE_KEY_JSON_PATH = None
-PRIVATE_KEY_JSON_CONTENTS = None
+PROJECT_ID = os.getenv('GOOGLE_CLOUD_PROJECT')
```

Comment:
https://pandas-gbq.readthedocs.io/en/latest/contributing.html are the docs for how to test this